resources/views/components/drop-down.blade.php

```diff
@@ -5,35 +5,29 @@
 
 props:
  - $title
- - $varian
+ - $variant
  - $slot
 
 UI components used:
  - button --}}
 
-<div
-  class="text-right relative"
-  x-data="{ open: false }"
->
+<div class="relative" x-data="{ open: false }">
```

Comment:
Hi @cole-flexsim these are the kind of changes about code style that I was referring to in my comment

Reply:
thanks! that way we can focus the PR on the actual feature